aiida/cmdline/commands/cmd_storage.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""`verdi storage` commands."""
+
+import click
+
+from aiida.cmdline.commands.cmd_verdi import verdi
+from aiida.cmdline.params import options
+from aiida.cmdline.utils import echo
+from aiida.common import exceptions
+
+
+@verdi.group('storage')
+def verdi_storage():
+    """Inspect and manage the internal storage."""
```

Comment:
This feels a bit "non-descript", I don't think a user would really know what this meant.

Perhaps something like

```suggestion
    """Inspect and manage stored data for a profile."""
```

Reply:
Done.